build_tools/cmake/iree_import_binary.cmake

```diff
@@ -31,7 +31,7 @@ include(CMakeParseArguments)
 function(iree_import_binary)
   cmake_parse_arguments(
     _RULE
-    ""
+    "OPTIONAL"
```

Comment:
nit: document this up in the "Parameters" comment block above

e.g.

> OPTIONAL: if set, log a warning instead of an error if the binary is not found